src/main/resources/l10n/JabRef_en.properties

```diff
@@ -1035,6 +1035,7 @@ required\ field\ is\ present=required field is present
 optional\ field\ is\ present=optional field is present
 unknown\ field\ is\ present=unknown field is present
 field\ is\ absent=field is absent
+Field\ does\ not\ exist\ in\ the\ editor!=Field does not exist in the editor!
```

Comment:
Avoid exclamation marks at the end of a sentence. They are more for screaming. Use a dot to end the sentence.